dom.bs

```diff
@@ -5776,8 +5776,8 @@ This and <a lt="other applicable specifications">other specifications</a> may de
 
 <ol>
  <li>
-  <p>If <var>inserted</var> is [=connected=], and it has a content attribute (<var>attr</var>) whose
-  value is not the empty string, then:</p>
+  <p>If <var>inserted</var> is [=connected=], and it has a content attribute (<var>attr</var>) named
+  [=Attr/local name=] "<code>nonce</code>" whose [=Attr/value=] is not the empty string, then:</p>
```

Comment:
Trying again in https://github.com/whatwg/dom/pull/436/commits/78db5fb120d10477b70f3f570ebea88def16d4bd. :)